runtime/doc/eval.txt

```diff
@@ -8631,13 +8653,16 @@ stdpath({what})					*stdpath()* *E6100*
 			:echo stdpath("config")
 
 
-str2float({expr})					*str2float()*
-		Convert String {expr} to a Float.  This mostly works the same
-		as when using a floating point number in an expression, see
-		|floating-point-format|.  But it's a bit more permissive.
-		E.g., "1e40" is accepted, while in an expression you need to
-		write "1.0e40".  The hexadecimal form "0x123" is also
-		accepted, but not others, like binary or octal.
+str2float({string} [, {quoted}])				*str2float()*
+ 		Convert String {string} to a Float.  This mostly works the
+ 		same as when using a floating point number in an expression,
+ 		see |floating-point-format|.  But it's a bit more permissive.
+ 		E.g., "1e40" is accepted, while in an expression you need to
+ 		write "1.0e40".  The hexadecimal form "0x123" is also
+ 		accepted, but not others, like binary or octal.
+ 		When {quoted} is present and non-zero then embedded single
+ 		quotes before the dot are ignored, thus "1'000.0" is a
+ 		thousand.
```

Comment:
Needs #14611 (which might be close to merging anyway?)

Reply:
Wait, I'm confusing it with `str2nr` -- this needs some other patch :P

Reply:
Yes, that's why I aspirationally included it (or rather didn't mess with the successful git merge).